source/web-service/flaskapp/routes/records.py

```diff
@@ -838,19 +838,26 @@ def entity_record_activity_stream(entity_id):
     limit = current_app.config["ITEMS_PER_PAGE"]
     total_pages = str(math.ceil(count / limit))
```

Comment:
The `str` cast on line `839` from some previous commit is no longer needed; nothing in scope, including the call to the `url_record()` helper function (which also takes an integer for its second argument), needs it to be a string; removing the `str` cast, and allowing `total_pages` to return to being an integer would allow the numeric comparison on line `841` to be clearer, and obviate the need to compare `total_pages` to the string representation of `"0"`.